tasks/oah-install.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Running Java version command
+  shell: "java --version"
+  register: java_output
+
+- name: Check if Java is installed
+  failed_when: '"Error" in java_output.stdout'
+
+- name: Checking Latest version of SonarQube.

```

Comment:
ERROR! conflicting action statements: shell, warn

![image](https://user-images.githubusercontent.com/58501088/190384414-9a343fc0-0369-476d-aecd-9dc7f3cc7f78.png)

Reply:
Removed the standalone failed_when task and changed the validating condition. Now working